catalog/proguard-rules.pro

```diff
@@ -0,0 +1,13 @@
+# This is generated automatically by the Android Gradle plugin.
+-dontwarn org.bouncycastle.jsse.BCSSLParameters
+-dontwarn org.bouncycastle.jsse.BCSSLSocket
+-dontwarn org.bouncycastle.jsse.provider.BouncyCastleJsseProvider
+-dontwarn org.conscrypt.Conscrypt$Version
+-dontwarn org.conscrypt.Conscrypt
+-dontwarn org.conscrypt.ConscryptHostnameVerifier
+-dontwarn org.openjsse.javax.net.ssl.SSLParameters
+-dontwarn org.openjsse.javax.net.ssl.SSLSocket
+-dontwarn org.openjsse.net.ssl.OpenJSSE
+-dontobfuscate
```

Comment:
I don't think we should use this `-dontobfuscate` otherwise there is probably no point to enable proguard.

Reply:
Actually I just want to use R8 for the shrinking/minify & optimisation and since we don't have things to hide it's simpler to debug release crashes wit this option

Reply:
Oups, I was thinking of -dontoptimize 🙈